md_api/src/main/java/com/wipro/ats/bdre/GetLineageQueryByProcessId.java

```diff
@@ -54,7 +54,10 @@ public GetLineageQueryByProcessId() {
         try {
             CommandLine commandLine = getCommandLine(params, PARAMS_STRUCTURE);
             String pid = commandLine.getOptionValue("sub-process-id");
-            LOGGER.debug("Instance exec id  is " + pid);
+            LOGGER.debug("Sub-Process id  is " + pid);
+            if(pid.equals("EMPTY")){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "EMPTY" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1132)